scripts/ci/make/pipeline_tasks

```diff
@@ -77,7 +77,14 @@ functional_tests() {
     activate_venv "${VENV}"
     hash tox || "${PYTHON}" -m pip install tox
     hash anchore-cli || "${PYTHON}" -m pip install -q anchorecli
-
+    if [[ "$CI" == "true" ]] && ! hash skopeo; then
```

Comment:
I generally don't mind scripts as long as they are kept super short and don't deal with too much logic. These lines below ensure `skopeo` is installed, but there are many problems with this approach and I am concerned it will start becoming a copy/pasta pattern repeated everywhere:

* `sudo` handling looks a bit odd because it is piping via `echo` so it requires `su -c` and then the command as a string. This decreases readability as there is no longer any highlighting.
* The `apt-get update` has to be done regardless if this is the first or tenth package being added. I anticipate this will not be the last dependency that requires installation and others will pile up on the `apt-get update`
* Enlarges the `functional_tests` to a point where it is no longer concise as it has to deal with dependencies.

This is fine for now, but it is important to raise awareness of the potential problems here.